docs/source/rpc.rst

```diff
@@ -51,8 +51,12 @@ RPC and RRef primitives
 This library provides primitives allowing users to create and modify references
 (RRefs) to remote data as well as remotely execute functions.
 
-.. automodule:: torch.distributed.rpc.api
-    :members:
+.. automodule:: torch.distributed.rpc
```

Comment:
I was trying to do something similar when working on the first version of this docs, but the downside with this approach is that if we add a new function, we'd have to manually remember to add to the doc here. Not sure how to get around that.

Reply:
I agree, that is indeed a problem for this approach, and we can figure out how to solve it later (Maybe remove `api.py` and move everything to `__init__.py`). As for now, let's make sure the doc exposes the correct path.